practices/tips_for_coders.md

```diff
@@ -0,0 +1,109 @@
+
+2. Googling is the most important skill
+
+We live in a world of information abundance.Also people are more interested in hiring people who can use the information constructively.
+
+3. Version Control is a must
+
+Version Control is probably the most important tool for a software developer. It saves, backs up your codebase, and also enables you to collaborate with other developers.
+
+4. One Completed Project is better than 100 Incomplete Ones
+
+As mentioned previously, people judge you based on what you’ve done, not what you plan to do. So put your best foot forward and have a few deployed projects, to ease up the process for the viewer.
+
+5. Self-taught Programmers have the best debugging skills
+
+Self-taught Programmers lack the opportunity of having someone help out whenever they face a problem, so after 100s of trial and error, they develop efficient ways to squash a bug.
+
+6. You don’t need the right answer, just a less wrong one
+
+People are so afraid of failing that they don’t even give things a shot. A failure is truly a failure, only if you don’t learn from it and keep repeating the same mistakes.
+
+7. Tutorials fail to teach you the most important skill
+
+The most crucial skill problem-solving, which comes only through trial and error. Tutorials give you a highly refactored final version of the code, so you miss out on seeing the skill in action.
+
+8. Learn what NOT to learn
+
+Learning every is simply not humanely possible. First test out the waters by trying out everything, then choose the technology that you feel the best for you.
+
+9. Consistency
+
+Nobody became a pro by putting in 20 hours in a day, but a lot of people did it by putting in 1 hour for 20 days. Your brain needs time to digest the information, you feed it.
+
+10. Open Source can help you level up
+
+Open Source is a great way to learn by building amazing Softwares and tools. But be prepared to be constantly learning as you would be required to be constantly adapting to new technologies. 
+
+## Why You Might Want The Tips For Coders
+
+A good coder is not made in a day, a week or a month.
+It takes much grinding and hardwork to polish your skills. These tips are just to give you a roadmap and help you out in your coding journey.
+
+## Problems The Tips For Coders Helps to Solve
+
+It helps beginners to start a smoother journey.
+It helps the experienced ones to become more efficient and have trust in themselves.
+
+These tips might seem very normal and basic but are missed out by most of us so consider these all just to remind you of the best practices you should follow.
+
+## How to Implement the Tips For Coders?
+
+Just follow them daily throughout your coding journey.
+Or feel free to come back and read them again to build more confidence.
+They could prove to be very motivating and guide you to the right path.
+
+## Common Pitfalls of the Tips For Coders
```

Comment:
I suggest changing the title and content to "Common Pitfalls of the Path to Become a Coder".